contrib/pg_tde/documentation/docs/setup.md

```diff
@@ -173,16 +155,36 @@ Here's how to do it:
         This setup is intended for development and stores the keys unencrypted in the specified data file.    
 
         ```sql
-        SELECT pg_tde_add_key_provider_file('provider-name','/path/to/the/keyring/data.file');
+        SELECT pg_tde_add_global_key_provider_file('provider-name','/path/to/the/keyring/data.file');
```

Comment:
default_key_provide also works

Reply:
Should we elaborate it here? Or just create a reference here to possible options once all those functions are documented?